scripts/mkcloud

```diff
@@ -479,6 +479,14 @@ function crowbar_register
     done
 }
 
+# setup an NFS server on the first lonely node
+function lonelynode_nfs_server
+{
+    local nfsserver=($(nodes ids lonely))
+    local mac=$(macfunc $nfsserver)
+    sshrun "nfsservermac=$mac adminip=$adminip onadmin_setup_nfs_server"
```

Comment:
better use the function that is meant to do this:

```
onadmin setup_nfs_server $mac
```

The $adminip variable is passed to the admin server anyway.